app/src/main/java/be/scri/fragments/MainFragment.kt

```diff
@@ -39,9 +42,34 @@ class MainFragment : ScribeFragment("Main") {
             }
         (requireActivity() as MainActivity).setActionBarButtonVisibility(false)
         callback.isEnabled = true
+        isArabicLanguage()
         return binding.root
     }
 
+    private fun isArabicLanguage(): Boolean {
+        val cornerTriangleImageView: ImageView = binding.cornerTriangle
+        val cornerCogImageView: ImageView = binding.cornerCog
+        val paramsTriangleLayout = binding.cornerTriangle.layoutParams as FrameLayout.LayoutParams
+        val paramsCogLayout = binding.cornerCog.layoutParams as FrameLayout.LayoutParams
+        val currentLocale =
+            requireActivity()
+                .resources.configuration.locales
+                .get(0)
+        val languageCode =
+            currentLocale
+                .language
+        if (languageCode == "ar") {
+            paramsTriangleLayout.gravity = Gravity.TOP or Gravity.START
```

Comment:
for gravity, I think we do not need to handle it, as it already handled in xml, it set to top|end so in RTL it will be automatically be on left of screen.